README.md

```diff
@@ -10,3 +10,29 @@ Please download, unzip, and copy the corresponding asset folder for your platfor
 * Android - `/android/bin`
 * Win32 - The win32 binaries are not included in the zip assets. They are available on download on [nuget](https://www.nuget.org/packages/Microsoft.PlayFab.PlayFabParty.Cpp.Windows).
 * Linux - `/x64/Release/libparty.so`
+
+### Using PlayFab Party in Production
```

Comment:
I think this section could be simplified/clarified to just a single warning block

```
> :warning: - **Pre-built binaries of PlayFabXPlatCppSDK included in release packages are intended only for use in the PartySamples**. These binaries are not intended for production consumption and you should instead build the XPlatCppSDK from source in your project
>
> On android, this includes the `libparty-android-release/<arch>/lib/libcppsdk.a`
>
> On iOS, this includes `PlayFabParty.framework-for-iOS-Release/PlayFabParty.framework-for-iOS-<arch>-Release/libplayfabxplatcppsdk.a`
```